movies_test.py

```diff
@@ -0,0 +1,140 @@
+"""Tests the functions of the code.py file"""
+
+import movies
+
+def customer_db():
+    """Define Customer database for testing"""
+    return {
+        "Bob" : {"movies":[],
+                 "cash" : 100.0},
+        "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+                 "cash" : 10.0},
+        "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+                  "cash" : 120.0},
+        "Ricci": {"movies": [],
+                  "cash" : 4.0}
+    }
+
+def movie_db():
+    """Define movie database for testing"""
+    return {
+        "Xmen" : {"title" : "Xmen 8: The Xmennening", "seats_avaliable" : 10},
+        "Bromance" : {"title" : "The Bromance", "seats_avaliable" : 20},
+        "Gigli" : {"title" : "Gigli: The Play: The Book: The movie", "seats_avaliable" : 102}
+    }
+
+def test_enough_money():
+    """Tests the enough money function"""
+    customers = customer_db()
+    bob = customers["Bob"]
+    ricci = customers["Ricci"]
+    assert not movies.enough_money(ricci)
+    assert movies.enough_money(bob)
+
+def test_charge():
+    """Tests charge function"""
+    customers = customer_db()
+    bob = customers["Bob"]
+    assert movies.enough_money(bob)
+    bobs_actual_cash = movies.charge(bob)["cash"]
+    bobs_expected_cash = 95.0
+    assert bobs_actual_cash == bobs_expected_cash
+    jim = customers["Jim"]
```

Comment:
let's get some whitespace in here to separate out the tests within each test - you should go ahead and do this for all of your tests - otherwise, it can be a bit muddled and harder to follow what is actually being tested